tests/test_core.py

```diff
@@ -30,10 +31,12 @@ def config(request, monkeypatch):
 def test_bigchain_class_default_initialization(config):
     from bigchaindb.core import Bigchain
     from bigchaindb.consensus import BaseConsensusRules
+    from bigchaindb.backend.connection import Connection
     bigchain = Bigchain()
-    assert bigchain.host == config['database']['host']
-    assert bigchain.port == config['database']['port']
-    assert bigchain.dbname == config['database']['name']
+    assert isinstance(bigchain.connection, Connection)
```

Comment:
I wasn't 100% sure as well, but I think it's OK for now. From my experience `asset something` is too loose. I prefer to have a stricter tests that will eventually fail, so we can fix it again in the proper way afterwards.